Equilobe.DailyReport.SL/DataService.cs

```diff
@@ -151,6 +151,9 @@ public void SetReportFromDb(JiraReport _report)
                 var reportSettings = db.BasicSettings.SingleOrDefault(r => r.UniqueProjectKey == _report.UniqueProjectKey);
                 _report.Settings = new BasicSettings();
                 reportSettings.CopyTo<BasicSettings>(_report.Settings);
+                reportSettings.InstalledInstance.CopyTo<InstalledInstance>(_report.Settings.InstalledInstance);
+                reportSettings.InstalledInstance.UserImages.CopyTo<ICollection<UserImage>>(_report.Settings.InstalledInstance.UserImages);
```

Comment:
Aici vei folosi `Include` cand vei trece pe structura modificata in pull requestul de la DRT-150.